lib/libzfs/libzfs_sendrecv.c

```diff
@@ -911,13 +911,20 @@ send_progress_thread(void *arg)
 	char buf[16];
 	time_t t;
 	struct tm *tm;
-	boolean_t firstloop = B_TRUE;
+	int err;
+
+	if (!pa->pa_parsable) {
+		(void) fprintf(stderr,
+		    "TIME       %s   %sSNAPSHOT %s\n",
+		    pa->pa_estimate ? "BYTES" : " SENT",
+		    pa->pa_verbosity >= 2 ? "   BLOCKS    " : "",
+		    zhp->zfs_name);
+	}
```

Comment:
No objection, but my guess is it was originally done this way so the header wouldn't be printed until after the sleep.